src/integration-test/resources/archives/workflow_0.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The file could be renamed more meaningful as "workflow_hello_world" ?

Reply:
I updated the code to remove this file.